tron/serialize/runstate/statemanager.py

```diff
@@ -60,7 +59,8 @@ class StateMetadata(object):
     RunState interface as Jobs and Services.
     """
     name = 'StateMetadata'
-    version = tron.__version_info__
+    # version = tron.__version_info__
+    version = (0, 7, 0, 0)
```

Comment:
Move it to tron/__init__.py maybe?

Reply:
`tron/__init__.py` is for actual tron release version, this is state schema version (effectively only the first component).